### PR TITLE
Add affinity for BackupPolicySpec

### PR DESCRIPTION
**Pull request by d-kuro**
refs: https://github.com/cybozu-go/moco/issues/413

### CODE REVIEW

**Comment by ymmt2005** [comment]
Could you also create an example manifest in `examples` directory?
https://github.com/cybozu-go/moco/tree/main/examples

**Comment by d-kuro** [comment]
@ymmt2005 Thanks review, sorry for late the reply.
Question, shouldn't it add anti-affinity by default?

https://github.com/cybozu-go/moco/pull/471/commits/57e56b4a99b087ea3e36832d2472be22fd4714d3

**Comment by ymmt2005** [comment]
@d-kuro 
Good point. I think it's a good idea to set an anti-affinity by default as long as its policy is `preferredDuringSchedulingIgnoredDuringExecution`.

@masa213f what do you think?

**Comment by masa213f** [comment]
@ymmt2005 @d-kuro 
Sorry for the late reply.
That's good idea to set the default anti-affinity of preferredDuringSchedulingIgnoredDuringExecution.

**Comment by d-kuro** [comment]
@ymmt2005 @masa213f 
Sorry for the delay in responding.
I added default anti-affinity with `preferredDuringSchedulingIgnoredDuringExecution` and update docs.
https://github.com/cybozu-go/moco/pull/471/commits/bf10dbf8fef64b53c99b2ca02c90b4bd83043eb2

File: pkg/constants/meta.go [hunk 1]
```diff
@@ -10,6 +10,8 @@ const (
 	LabelAppCreatedBy = "app.kubernetes.io/created-by"
 	AppCreator        = "moco"
 
+	LabelHostName = "kubernetes.io/hostname"
```
Comment (ymmt2005) [inline pkg/constants/meta.go:13]: I think we can use `corev1.LabelHostname`.
https://github.com/kubernetes/api/blob/3a2d6b5bb7f84daa2125131865eb81b151448d5b/core/v1/well_known_labels.go#L20

**Review by ymmt2005 — CHANGES_REQUESTED** [review]

**Review by masa213f — DISMISSED** [review]
LGTM. Please fix https://github.com/cybozu-go/moco/pull/471#discussion_r1048036669

File: pkg/constants/meta.go (hunk 1 above)
Comment (d-kuro) [reply pkg/constants/meta.go]: Thanks! I fixed: https://github.com/cybozu-go/moco/pull/471/commits/bef9388871be165040b78b4d277169df335aee70

File: e2e/mycluster.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+apiVersion: moco.cybozu.com/v1beta1
```
Comment (masa213f) [inline e2e/mycluster.yaml:1]: Do we need this file?

**Review by masa213f — CHANGES_REQUESTED** [review]

File: e2e/mycluster.yaml (hunk 2 above)
Comment (d-kuro) [reply e2e/mycluster.yaml:1]: I certainly may not need it.
I will remove it.

Comment (d-kuro) [reply e2e/mycluster.yaml:1]: Sorry, I committed another unnecessary file.
I remove it.

**Review by ymmt2005 — DISMISSED** [review]

File: e2e/mycluster.yaml (hunk 2 above)
Comment (d-kuro) [reply e2e/mycluster.yaml]: I deleted the file and squashed the commit.
https://github.com/cybozu-go/moco/pull/471/commits/4548faf31bb4d5341048be6d40add60478a264ce

**Review by masa213f — APPROVED** [review]
LGTM

**Pull request merged**